README.md

```diff
@@ -185,6 +185,33 @@ You can get an equivalent of the Command Line Tool using the Python3 executable
 $ python3 -m pyshacl
 ```
 
+## Errors
+Under certain circumstances pySHACL can produce a `Validation Failure`. This is a formal error defined by the SHACL specification and is required to be produced as a result of specific conditions within the SHACL graph.
```

Comment:
Would this be an appropriate link for "Validation failures"?

https://www.w3.org/TR/shacl/#failures

I mostly remembered the difference between "Validation failure" and [`sh:ValidationResult`](https://www.w3.org/TR/shacl/#result), but still felt a need to check.  It might be worth linking the failures section in case someone didn't know about this term nuance yet.

Reply:
Yes, that is the correct link. I agree I should probably put in a link to the relevant section in the spec.

Note, these descriptions are not new, they were/are already in the README file, this diff is simply moving the errors section to make a place for the Content for the SPARQL Remote Store.

Reply:
I did catch that this diff was moved text.  Eventually. Unfortunately, my curiosity was quicker than my scrolling.